### ISSUE TITLE
Request: ASM Export, include the shift / $d018 pattern in the export

### ISSUE THREAD

**Issue by wbochar** [nurpax/petmate#170]
If you are working on multiple screens with different ABC vs abc on export you can not tell from the data which mode it is in. So maybe along with the colours (BG, Border) include the ABC/abc mode

**Comment by nurpax** [comment]
Makes sense.  Including one more byte in the output will break binary compatibility with existing asm projects tho.  Currently the binary format is the same that Marq's editor exports.  But wouldn't be a big deal to add a configuration option to the export dialog that can be used to select the layout.

A better format for use in assembly projects is the JSON format.  At least if your assembler is flexible enough to read and transform such data.  Here's how I use Petmate .json files in my projects: https://nurpax.github.io/posts/2019-06-06-c64jasm-content-example.html

**Comment by wbochar** [comment]
Yeah, I do a lot of Cordova/JS... It's nice to have JSON flexibility and the auto compile :)
